cmd/program/program.go

```diff
@@ -211,10 +240,25 @@ func (p *Project) CreateMainFile() error {
 
 		err = p.CreateFileWithInjection(internalDatabasePath, projectPath, "database.go", "database")
 		if err != nil {
-			log.Printf("Error injecting server.go file: %v", err)
+			log.Printf("Error injecting database.go file: %v", err)
 			cobra.CheckErr(err)
 			return err
 		}
+
+		
+	}
+
+	// cretae correct docker compose for the selected driver
+	if p.DBDriver != "none" && p.DBDriver != "sqlite" {
```

Comment:
We might want to handle the case where they choose sqlite and we reply saying we cannot create a docker-compose instance, or something like that